### PR TITLE
Less support

### PR DESCRIPTION
**Pull request by ryanbriscall**
/cc: @coliff

### CODE REVIEW

**Comment by coliff** [comment]
Hi there. Why am I being CCed?

**Comment by SalahAdDin** [comment]
:+1: for sass support :D

File: source/helpers/jquery.fancybox-buttons.core.less [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+  display: block;
+  width: 166px;
+  height: 30px;
+  margin: 0 auto;
+  padding: 0;
+  list-style: none;
+  border: 1px solid #111;
+  border-radius: 3px;
+  -webkit-box-shadow: inset 0 0 0 1px rgba(255,255,255,.05);
+     -moz-box-shadow: inset 0 0 0 1px rgba(255,255,255,.05);
```
Comment (hebbet) [inline source/helpers/jquery.fancybox-buttons.core.less]: would move that to mixins.

**Comment by SalahAdDin** [comment]
sass support please.

**Comment by christianmagill** [comment]
+1 for Sass support

**Comment by hebbet** [comment]
File a new bug for sass.

**Pull request closed without merging**